content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -0,0 +1,171 @@
+  for when this field would be populated is if a company has several office locations. The typical
+  office location where the vehicle is located would be the location entered.
+- :guilabel:`Company`: Select the company that the vehicle will be used for and associated with from
+  the drop-down menu, or type in a new company and click either :guilabel:`Create` or
+  :guilabel:`Create and Edit`.
+
+.. important::
+   Creating a new company may cause a subscription price change depending on the current plan. Refer
+   to `Odoo's pricing plan <https://www.odoo.com/pricing-plan>`_ for more details.
+
+.. image:: new_vehicle/new-vehicle-type.png
+   :align: center
+   :alt: The new vehicle form, showing the vehicle tax section.
+
+Tax Info tab
+------------
+
+Fiscality
+~~~~~~~~~
+
+- :guilabel:`Horsepower Taxation`: Enter the amount that is taxed based on the size of the vehicles
+  engine. This is determined by local taxes and regulations, and varies depending on the location.
+  It is recommended to check with the accounting department to ensure this value is correct.
+- :guilabel:`Disallowed Expense Rate`: This is the amount of non-deductible expenses for the
+  vehicle. This amount is not counted towards any deductions on a tax return or as an allowable
+  expense when calculating taxable income. It is recommended to check with the accounting department
+  to ensure the value(s) entered are correct.
+
+  - :guilabel:`Start Date`: Enter the :guilabel:`Start Date` and :guilabel:`(%) Percentage` for when
+    the :guilabel:`Disallowed Expense Rate` value goes into effect. Click :guilabel:`Add a line` to
+    enter a date. Click on the blank line to display a calendar. Select the date by navigating to
+    the correct month and year using the :guilabel:`⬅️ (left arrow)` and :guilabel:`➡️ (right
+    arrow)` icons, then click on the specific day. Enter the percentage that is disallowed in the
+    :guilabel:`% (percent)` field to the right of the date. The percentage should be entered in an
+    XX.XX format. Repeat this for all entires needed.
+
+Contract
+~~~~~~~~
+
+- :guilabel:`First Contract Date`: Select the start date for the vehicle's first contract using
+  the drop-down calendar. Typically this is the day the vehicle is purchased or leased.
+- :guilabel:`Catalog Value (VAT Incl.)`: Enter the MSRP (Manufacturer's Suggested Retail Price) for
+  the vehicle at the time of purchase or lease.
+- :guilabel:`Purchase Value`: Enter the purchase price or the value of the lease for the vehicle.
+- :guilabel:`Residual Value`: Enter the current value of the vehicle.
+
+.. note::
+   The values listed above will affect the accounting department. It is recommended to check with
+   the accounting department for more information and/or assistance with these values.
+
+.. image:: new_vehicle/new-vehicle-tax.png
+   :align: center
+   :alt: The new vehicle form, showing the vehicle tax section.
+
+Model tab
```

Comment:
Will this information populate after selecting the model type? This is documented already in your other doc, we should be anchoring to that doc if that's the case. https://github.com/odoo/documentation/pull/5763

Reply:
OMG TIM- NOW I understand!!! Because you CAN enter the info when making a new model, and if you select the new saved model it DOES auto-populate! I was thinking of this form as in terms of making a new car, everything new- and a lot of the info they ask for is identical. BUT if they select a model that is already in the system, then they will have a lot of the fields auto-populated. OK, now I know which should be the main one and which should direct to the other doc. Thank you- that took me a while to wrap my head around!